ferrocene/doc/user-manual/src/self-test/using.rst

```diff
@@ -0,0 +1,47 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Using
+=====
+
+``ferrocene-self-test`` ensures that the Ferrocene toolchain is installed properly.
+
+The most easy way to use the tool is with `CriticalUp <https://criticalup.ferrocene.dev>`_:
+
+.. code::
+
+   criticalup run ferrocene-self-test
+
+If, instead, the toolchain was installed manually, the tool can be run directly:
+
+.. code::
+
+   <toolchain directory root>/bin/ferrocene-self-test
+
+.. note::
+
+   - The tool can only be used with the toolchain it is distributed with
```

Comment:
```suggestion
   - The tool should only be used with the toolchain it is distributed with
```

Reply:
there is FST_007 error code when user tries using the wrong version, which happens before the tool goes far
```
❯ bin/ferrocene-self-test
    Info: using sysroot /Users/tshepang/repos/ferrocene/main
   Error: expected release of rustc to be 1.79.0, found 1.82.0-nightly
    Note: the error code is FST_007